pkg/agent/core/ngt/handler/grpc/handler.go

```diff
@@ -21,10 +21,11 @@ import (
 	"context"
 	"fmt"
 	"strconv"
+	"sync"
 
-	agent "github.com/vdaas/vald/apis/grpc/agent/core"
-	"github.com/vdaas/vald/apis/grpc/gateway/vald"
-	"github.com/vdaas/vald/apis/grpc/payload"
+	agent "github.com/vdaas/vald/apis/grpc/v1/agent/core"
+	"github.com/vdaas/vald/apis/grpc/v1/payload"
+	vald "github.com/vdaas/vald/apis/grpc/v1/vald"
```

Comment:
I think this sentence is redundant. Is this on purpose?

Reply:
fixed